### ISSUE TITLE
Help text on Windows refers to the tool name as "cli" although it's installed as "markdownlint"

### ISSUE THREAD

**Issue by DavidAnson** [igorshubovych/markdownlint-cli#2]
This is a minor issue, but potentially confusing for novices. I'd suggest that "Usage:" should say "markdownlint" instead of "cli":

```
D:\T>markdownlint --help

  Usage: cli [options] <files>

  MarkdownLint Command Line Interface

  Options:

    -h, --help                 output usage information
    -V, --version              output the version number
    -c, --config [configFile]  Configuration file
```

As an *extremely* minor point, the 'O's of "Output" might be capitalized and the 'V' of "-V" lower-cased for consistency with the other switches.

**Comment by igorshubovych** [comment]
It is interesting, I see it as correctly:
http://take.ms/ylei3

Have you installed it globally (-g)? As far as I see you are using Windows.

**Comment by DavidAnson** [comment]
Commander attempts to infer the tool name if it is not passed in - see `Command.prototype.parse`.

On Windows (including when installed globally), argv at that point looks like:

```
[ 'C:\\Program Files\\nodejs\\node.exe',
  'd:\\T\\markdownlint-cli\\cli.js' ]
```

Therefore, commander infers the name "cli".

One fix might be to rename `cli.js` to `markdown.js`. Another might be something like the following:

```js
var pkg = require('./package');
var commander = require('commander');
var program = new commander.Command(pkg.name.replace(/-.*$/, ''));
```

If you like, I can send a pull request - or feel free to make the change yourself if you like it.

**Comment by igorshubovych** [comment]
Thanks, @DavidAnson.

For now I just renamed it.
But I don't like it already, if i will see more issues like this, i will rewrite it using [meow](https://github.com/sindresorhus/meow), argparse, minimist anything like that instead of commander.
